package.json

```diff
@@ -53,14 +53,14 @@
         "dev": "encore dev",
         "watch": "encore dev --watch",
         "build": "encore production --progress",
-        "check-coding-standards-stylelint": "stylelint --config=.stylelintrc.js 'bundles/kontrolgruppen-core-bundle/Resources/assets/**/*.scss'",
+        "check-coding-standards-stylelint": "stylelint --config=.stylelintrc.js 'vendor/kontrolgruppen/core-bundle/Resources/assets/**/*.scss'",
```

Comment:
I think, personally, that this should be moved to the `core-bundle`'s own `package.json`, but we'll leave it like this for now.